.github/workflows/load-balancers.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [python]
+        target: [nodejs]
```

Comment:
target here replaces python with nodejs while in the golang PR you added golang to the existing python target